Remove agent-config shortcode (#31905)
DataDog/documentation: content/en/agent/logs/_index.md

{{% tab "TCP/UDP" %}}

> To collect the sender IP address and associate with the log message payload, add the following configuration to your `datadog.yaml` file:

Reviewer: ```suggestion
To capture the sender IP address and include it in the log message payload, add the following configuration to your `datadog.yaml` file:
```
I know this wording was there before but it was a little confusion so suggest to reword a bit for clarity :)
